guides/common/modules/proc_migrating-project-or-smartproxy-using-backup.adoc

```diff
@@ -17,7 +17,7 @@ endif::[]
 . Perform a full backup of your {ProjectServer} or {SmartProxy}.
 This is the source {EL}{nbsp}8 server that you are migrating.
 For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin#Performing_a_Full_Backup_admin[Performing a full backup of {ProjectServer} or {SmartProxyServer}] in _{AdministeringDocTitle}_.
-. Deploy a system with {EL}{nbsp}9.
+. Deploy a {EL}{nbsp}9 system with the same hostname and configuration as the source server.
```

Comment:
```suggestion
. Deploy a system with {EL}{nbsp}9 and the same hostname and configuration as the source server.
```
To avoid the "an Enterprise Linux" vs. "a RHEL" problem.